### PR TITLE
Edit scores in matchups

### PR DESCRIPTION
**Pull request by andygw**

### CODE REVIEW

**Comment by andygw** [comment]
The show page has a button to set the score
image

File: app/policies/tournament_policy.rb [hunk 1]
```diff
@@ -5,6 +5,7 @@ def resolve
       scope.all
     end
 
+    # TODO: Should this be in scope?
```
Comment (squush) [inline app/policies/tournament_policy.rb:8]: I believe this is not actually necessary, it is basically redundant with the scope.all thing

File: app/views/tournaments/show.html.erb [hunk 2]
```diff
@@ -1,8 +1,46 @@
...
+    <p>Number of winners:
+      <%= @tournament.number_of_winners.nil? ? "unknown" : @tournament.number_of_winners %>
+    </p>
+    <p>Pairing system: <%= @tournament.pairing_system.capitalize %></p>
+  </div>
+
+  <div class="mb-3 border-bottom border-3">
+    <p>Players</p>
+    <ul>
+      <% @tournament.players.each do |player| %>
```
Comment (squush) [inline app/views/tournaments/show.html.erb:17]: Looks good, but I have not yet scraped xtables to get all players of a tournament, so this will break

**Review by squush — CHANGES_REQUESTED** [review]

File: app/views/tournaments/show.html.erb (hunk 2 above)
Comment (andygw) [reply app/views/tournaments/show.html.erb:17]: This will work with my seed tournament

**Review by squush — APPROVED** [review]

**Pull request merged**